tests/e2e/statefulsets.go

```diff
@@ -85,13 +86,26 @@ var _ = ginkgo.Describe("statefulset", func() {
 				*metav1.NewDeleteOptions(0))).NotTo(gomega.HaveOccurred())
 		}
 		nimbusGeneratedK8sVmPwd = GetAndExpectStringEnvVar(nimbusK8sVmPwd)
+		svcMasterPswd = GetAndExpectStringEnvVar(svcMasterPassword)
```

Comment:
Can we make this as a local variable in test required instead of a global variable, as there is only 1 test that requires this variable.